core/src/main/scala/org/apache/spark/ui/JettyUtils.scala

```diff
@@ -79,6 +79,9 @@ private[spark] object JettyUtils extends Logging {
     val allowFramingFrom = conf.getOption("spark.ui.allowFramingFrom")
     val xFrameOptionsValue =
       allowFramingFrom.map(uri => s"ALLOW-FROM $uri").getOrElse("SAMEORIGIN")
+    val xXssProtectionValue = conf.getOption("spark.ui.xXssProtection")
```

Comment:
Please use `ConfigEntry` for newly added configurations, you could refer to `org.apache.spark.internal.config`.

Reply:
@jerryshao I am not sure if that will add much value in this context apart from following best practices.

Reply:
It follows new code convention, newly added configurations is better to change to that way.